index.php

```diff
@@ -1,12 +1,44 @@
 <?php
 header("Access-Control-Allow-Origin: *");
+
+/**
+ * Deps
+ */
+
+use \Psr\Http\Message\ServerRequestInterface as Request;
+use \Psr\Http\Message\ResponseInterface as Response;
+
+require './vendor/autoload.php';
+
+/**
+ * HTTP Handlers
+ */
+
+$app = new \Slim\App;
+
+$app->post('/api/v1/upload-session', function (Request $request, Response $response, array $args) {
+    $json = json_decode($request->getBody(), true);
+
+    // TODO: Persist upload session data
```

Comment:
@brunoblock @pandora23 
Will this be shared between nodes? Should I persist this in something like Redis?